src/services/activityReports.test.js

```diff
@@ -231,4 +235,26 @@ describe('Activity Reports DB service', () => {
       expect(foundReport.resourcesUsed).toBe('test');
     });
   });
+
+  describe('possibleRecipients', () => {
+    it('retrieves correct recipients in region', async () => {
+      const region = 17;
+      const recipients = await possibleRecipients(region);
+
+      expect(recipients.grants.length).toBe(1);
+    });
+
+    it('retrieves no recipients in empty region ', async () => {
+      const region = 100;
+      const recipients = await possibleRecipients(region);
+
+      expect(recipients.grants.length).toBe(0);
+    });
+
+    it('retrieves all recipients when not specifying region ', async () => {
```

Comment:
What is the use case for returning recipients without a filtering region?

Reply:
I left that possibility there for two things:
1. In case that functionality was still being used with the `regionId` param (I can double check to make sure thats not the case anywhere and make `regionId` mandatory.
2. In case we want to fetch all recipients without worrying of region.

Reply:
Based on https://gsa-tts.slack.com/archives/C017RJ4GEET/p1614609580004100?thread_ts=1614606923.002700&cid=C017RJ4GEET this is fine to leave in